lib/evmone/instructions_calls.cpp

```diff
@@ -180,6 +180,66 @@ evmc_status_code create_impl(StackTop stack, ExecutionState& state) noexcept
     return EVMC_SUCCESS;
 }
 
+evmc_status_code create3(StackTop stack, ExecutionState& state, code_iterator& pos) noexcept
+{
+    if (state.in_static_mode())
+        return EVMC_STATIC_MODE_VIOLATION;
+
+    const auto initcontainer_index = uint8_t{pos[1]};
+
+    const auto endowment = stack.pop();
+    const auto input_offset_u256 = stack.pop();
+    const auto input_size_u256 = stack.pop();
+    const auto salt = stack.pop();
+
+    stack.push(0);  // Assume failure.
+    state.return_data.clear();
+    state.deploy_container.reset();
+
+    if (!check_memory(state, input_offset_u256, input_size_u256))
+        return EVMC_OUT_OF_GAS;
+
+    const auto input_offset = static_cast<size_t>(input_offset_u256);
+    const auto input_size = static_cast<size_t>(input_size_u256);
+
+    if (state.msg->depth >= 1024)
+        return EVMC_SUCCESS;  // "Light" failure.
+
+    if (endowment != 0 &&
+        intx::be::load<uint256>(state.host.get_balance(state.msg->recipient)) < endowment)
```

Comment:
Hm why is this recipient and not sender? I know `create_impl` does the same but naming is counter intuitive.